services/src/main/java/org/keycloak/protocol/oidc/OIDCLoginProtocolService.java

```diff
@@ -273,6 +279,13 @@ public Object revoke() {
         return endpoint;
     }
 
+    @Path("par")
+    public Object par() {
```

Comment:
Can we avoid changes in the OIDCLoginProtocolService and use the extensions approach instead? For example see CibaRootEndpoint and the classes, which it inherits from (OIDCExtProvider, OIDCExtProviderFactory, EnvironmentDependentProviderFactory)

Reply:
@mposolda I see, I will follow the way of implementing newly introduced custom endpoint in `CibaRootEndpoint` and avoid modifying `OIDCLoginProtocolService` directly.

Reply:
Thanks